.github/workflows/library-and-framework-list-validation.yml

```
@@ -0,0 +1,16 @@
+name: "Validate library and framework list json"
+
+on: [push, pull_request]
+
+jobs:
+  validate-library-and-framework-list-json:
+    name: "📋 Checks if the json file is written according to the existing schema"
+    runs-on: "ubuntu-20.04"
+    steps:
+      - name: "☁️ Checkout repository"
+        uses: actions/checkout@v2
```

```suggestion
        uses: actions/checkout@v3
```
we need to use `v3` everywhere: https://github.blog/changelog/2022-09-22-github-actions-all-actions-will-begin-running-on-node16-instead-of-node12/